src/cloudant/account.py

```diff
@@ -393,22 +393,34 @@ def _usage_endpoint(self, endpoint, year=None, month=None):
 
         :param str endpoint: Cloudant usage endpoint.
         :param int year: Year to query against.  Optional parameter.
-        Defaults to None.  If used, it must be accompanied by the ``month``.
-        :param int month: Month to query against. Optional parameter.
-        Defaults to None.  If used, it must be accompanied by the ``year``.
-        """
-        if (year is not None) and (month is not None):
-            endpoint = posixpath.join(endpoint, str(year), str(month))
-        elif (year is not None) or (month is not None):
-            raise CloudantException(
-                (
-                    "must supply both year and month "
-                    "to usage endpoint: {0}"
-                ).format(endpoint)
-            )
-        resp = self.r_session.get(endpoint)
-        resp.raise_for_status()
-        return resp.json()
+            Defaults to None.  If used, it must be accompanied by ``month``.
+        :param int month: Month to query against that must be an integer
+            between 1 and 12. Optional parameter. Defaults to None.
+            If used, it must be accompanied by ``year``.
+        """
+        err = None
+        if year is None and month is None:
+            resp = self.r_session.get(endpoint)
+        else:
+            try:
+                if (year and int(year) > 0) \
+                        and (month and int(month) in range(1, 13)):
+                    resp = self.r_session.get(
+                        posixpath.join(
+                            endpoint, str(int(year)), str(int(month)))
+                    )
+                else:
+                    err = 'Invalid year and/or month supplied.  ' \
+                          'Found: year - {0}, month - {1}'.format(year, month)
+            except ValueError:
+                err = 'Invalid year and/or month supplied.  ' \
+                      'Found: year - {0}, month - {1}'.format(year, month)
+
```

Comment:
I think the point behind using the try/except block was to simplify the `if` logic by catching exceptions from the call to the `int()` built-in.  That being the case, you should change the except to catch not only `ValueError` but also `TypeError` which is what gets raised if either year or month are None.  As in `except (ValueError, TypeError):`.

Also for line continuation on statements don't use `\`.  Instead use parentheses.  The `if` statement should now fit on a single line once you remove the explicit check for `year` and `month` but you can change the `err` setting statements to: 

```python
err = ('Invalid year and/or month supplied.  '
    'Found: year - {0}, month - {1}').format(year, month)
```